Git-Remove-Submodule.md

```diff
@@ -0,0 +1,16 @@
+Removing a submodule is useful when it is no longer required. The steps below outline the removal of a submodule.
```

Comment:
We don't have an article for it. Later it can be created and linked. If someone is looking how to remove one, it is very likely they know what it is or enough about it already anyways.